_episodes/13-looking-up-data.md

```diff
@@ -162,5 +162,5 @@ As it returns the whole row for each match, you can use a transformation to extr
 
 You can use this function to compare the contents of two OpenRefine projects, or to use data between the two projects.
 
-The [VIB-Bits extension](https://www.bits.vib.be/index.php/software-overview/openrefine) adds a number of very useful functions to OpenRefine.
+The [VIB-Bits extension](https://data.bits.vib.be/hidden/g7dt6RjuUTU421dY2CwrGePGX/vib-bits.zip) ([manual](http://data.bits.vib.be/hidden/g7dt6RjuUTU421dY2CwrGePGX/OpenRefine%20VIB-BITS%20plugin.pdf)) adds a number of very useful functions to OpenRefine.
```

Comment:
I wonder if we should just point to the OpenRefine download page so that we don't have to independently keeps these links up to date? (I'm happy either way tbh, just wonder if it might be lower maintenance for the lesson to point at the download page)

Reply:
Yes, true, in particular these links don't look stable :-). Since the Extensions section right above already links to the downloads page, I've just removed the links here.